compute/admin_guide/waas/deploy_waas/fragments/add-app-policy.adoc

```diff
@@ -0,0 +1,122 @@
+. Select a WAAS rule to add an App in.
+
+. Click *Add app*.
+
+. In the *App Definition* tab, enter an *App ID*.
++
+NOTE: The combination of *Rule name* and *App ID* must be unique across In-Line and Out-Of-Band WAAS policies for Containers, Hosts, and App-Embedded.
```

Comment:
@Pubs-MV This [Note](https://redlock.atlassian.net/browse/CWP-42473?focusedCommentId=597649) is added here once, and will reflect in all pages.